src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/DefaultHttpMessageHandlerBuilderTest.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Net.Http;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.Metrics;
```

Comment:
Do we have any tests for published `System.Net.Http` metrics? the tests here are intergrating the IMeterFactory but no tests for testing it is working.

Reply:
It doesn't have metrics yet.